public/editor/scripts/editor/js/fc/project-rename.js

```diff
@@ -117,28 +117,30 @@ define(function(require) {
   }
 
   function save(context) {
-    context.saveButton.text("{{ renameProjectSavingIndicator }}");
-
-    persist.call(context, context.titleBar.val(), function(err) {
-      if(err) {
-        console.error("[Bramble] Failed to rename the project with ", err);
-        return;
-      }
+    // Do not apply save if project title remains the same
+    if (context.titleBar.val() === Project.getTitle()) {
+      editingComplete(context);
+    } else {
+      context.saveButton.text("{{ renameProjectSavingIndicator }}");
```

Comment:
does this template string actually get replaced? I don't remember how or why this worked.. (if you don't know that's fine, just trying to remember what magic happened here)